packages/interface/src/lib/eas/registerSchemas.ts

```diff
@@ -15,6 +15,8 @@ This means applications and profiles share the same schema and are differentiate
 Run: npx tsx src/lib/eas/registerSchemas
 */
 
+/** TODO: Not sure if we still need this file or not */
```

Comment:
I think we never use the registerSchemas, from what I recall this was from easy rpgf

Reply:
I remove most of the schema, but considered maybe we still need the voter schema (if using EAS Gatekeeper) so I kept it.